_docs-sources/guides/production-framework/ingredients/self-service/intro.md

```diff
@@ -0,0 +1,9 @@
+# Self-Service
+
+The next step is to take all the previous ingredients—the Service Catalog, the CI / CD pipeline, etc—and expose it to
+your developers in a self-service format.
+
+![Gruntwork Service Catalog](/img/guides/production-framework/self-service.png)
```

Comment:
I don't think this image should be shown _quite yet_. The customer hasn't seen this UI from Gruntwork and right now, they can't because it doesn't exist. 

I understand where you're going with this - but the customer has no such context. I think we need to find a better way/image

Reply:
Yea, I struggled with this, as we don't have a good image for this. Replaced in https://github.com/gruntwork-io/docs/pull/145/commits/9ed3ed3f82034a4cf151f0123e017401a707b590, but that isn't ideal either.